client/public/runner.html

```diff
@@ -114,6 +137,8 @@
         style.textContent = state.css;
         document.head.appendChild(style);
 
+        await loadExternalScripts(state.html);
+
         document.body.innerHTML = "";
         setHTML(document.body, state.html);
```

Comment:
Blocking here for the scripts should do the trick. Not using a promise all because we're not super close to real behavior anyhow. We'll move to a different and better solution later this year. In this case we don't need the load external scripts.
```suggestion
        setHTML(document.body, state.html);

        for (const script of document.querySelectorAll("script[src]")) {
          await (new Promise((resolve) => {
            script.addEventListener("load", () => resolve());
            script.addEventListener("error", () => resolve());
          }))
        }
```

Reply:
Oh, this is more laconic indeed! I've updated PR.